src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -28,12 +28,19 @@ internal class DpgOutputLibraryBuilder
         private readonly string _defaultNamespace;
         private readonly string _libraryName;
 
+        private readonly Dictionary<(string Namespace, string Name), Type> _azureCoreTypeMapping;
+
         public DpgOutputLibraryBuilder(InputNamespace rootNamespace, SourceInputModel? sourceInputModel)
         {
             _rootNamespace = rootNamespace;
             _sourceInputModel = sourceInputModel;
             _defaultNamespace = Configuration.Namespace ?? rootNamespace.Name;
             _libraryName = Configuration.LibraryName ?? rootNamespace.Name;
+            _azureCoreTypeMapping = new()
+            {
+                [("Azure.Core.Foundations", "Error")] = typeof(Azure.ResponseError),
+                [("Azure.Core.Foundations", "InnerError")] = typeof(Azure.ResponseError).Assembly.GetType("Azure.ResponseInnerError", true)!
```

Comment:
Can we have a config file for this, so if we found new types, we don't need to release a new codegen.
And can we use typeof(Azure.ResponseInnerError)) directly?

Reply:
We cannot use `typeof(Azure.ResponseInnerError))` because it is an internal class, I have to get it in this way.

In the future this map should be automatically built from the dependencies (maybe we could read it from the attributes on classes defined in `Azure.Core`.

Reply:
We no longer  need this therefore I already removed this part.